www/src/providers/Auth0ProviderWithHistory.tsx

```diff
@@ -17,7 +17,7 @@ const Auth0ProviderWithHistory: FC<Auth0ProviderWithHistoryProps> = ({ children
     };
 
     return (
-        <Auth0Provider domain={domain} clientId={clientId} redirectUri={window.location.origin} onRedirectCallback={onRedirectCallback}>
+        <Auth0Provider domain={domain} clientId={clientId} redirectUri={window.location.origin} onRedirectCallback={onRedirectCallback} audience={config.server.audience} scope={'call:ping'}>
```

Comment:
Fine for now, but those scopes should really become a TypeScript Enum.

Also question, eventually will the scopes this component needs to claim basically just be all possible scopes?

Reply:
Yes that's what I imagined, the returned scopes is an intersection of the scopes here and what the user actually has. If the user doesn't specify any scope in the beginning, we'll need to spawn another login or consent page to re-fetch the scope in the token

Reply:
Sounds good.